specs/binding-kafka.spec/src/main/scripts/io/aklivity/zilla/specs/binding/kafka/streams/application/describe.cluster/cluster.brokers.info/client.rpt

```diff
@@ -0,0 +1,55 @@
+#
+# Copyright 2021-2023 Aklivity Inc.
+#
+# Aklivity licenses this file to you under the Apache License,
+# version 2.0 (the "License"); you may not use this file except in compliance
+# with the License. You may obtain a copy of the License at:
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+
+connect "zilla://streams/app0"
+    option zilla:window 8192
+    option zilla:transmission "half-duplex"
+
+write zilla:begin.ext ${kafka:beginEx()
+                               .typeId(zilla:id("kafka"))
+                               .request()
+                                 .describeCluster()
+                                    .includeAuthorizedOperations("false")
+                                    .timeout(0)
+                                 .build()
+                               .build()}
+
+connected
+
+read zilla:begin.ext ${kafka:matchBeginEx()
+                               .typeId(zilla:id("kafka"))
+                               .response()
+                                 .describeCluster()
+                                    .throttle(0)
+                                    .error(0)
+                                    .clusterId("cluster-0")
+                                    .controllerId(0)
+                                    .broker()
+                                       .brokerId(0)
+                                       .host("localhost0")
```

Comment:
Let's use more meaningful broker ids and host names, like `0xb1` and `broker-1.internal.net` or similar.
I think we might have some examples of that in other scripts.